core/templates/domain/exploration/exploration-summary.model.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Frontend domain model for exploration summary.
+ */
+
+export interface ExplorationSummaryBackendDict {
+  'category': string;
+  'community_owned': boolean;
+  'last_updated_msec': number;
+  'id': string;
+  'created_on': number;
+  'language_code': string;
+  'objective': string;
+  'status': string;
+  'thumbnail_bg_color': string;
+  'thumbnail_icon_url': string;
+  'title': string;
+  'node_count': number;
+  'tags': string[];
+  /*
+  'owner_ids': ;
+  'editor_ids': ;
+  'viewer_ids': ;
+  'contributor_ids': ;
+  'contributors_summary': ;
+  'version': ;
+  'collection_model_created_on': ;
+  'collection_model_last_updated': ; */
```

Comment:
Looking at the backend code, it seems like there are a few missing keys and some differences, can you please take a look?

https://github.com/oppia/oppia/blob/a77bc93340cd2cfb9f538e4a4edf94b80ced68a3/core/domain/summary_services.py#L386-L411

Reply:
I made the changes, but now I am getting a wired error when trying to push:
<img width="1199" alt="Screenshot 2020-09-27 at 18 03 42" src="https://user-images.githubusercontent.com/10097737/94369743-e0935a80-00eb-11eb-8e22-ee4de6bde9d2.png">